bundles/org.openhab.core.io.websocket/src/main/java/org/openhab/core/io/websocket/audiopcm/PCMWebSocketStreamIdUtil.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.io.websocket.audiopcm;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.security.SecureRandom;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Utils to read/write the audio packets send though the websocket binary protocol.
+ *
+ * @author Miguel Álvarez Díez - Initial contribution
+ */
+@NonNullByDefault
+public class PCMWebSocketStreamIdUtil {
+    /**
+     * Packet header length in bytes:
+     * 2 for id
+     * 4 for sample rate as int little-endian
+     * 1 for bitDepth
+     * 1 for channels
+     */
+    public static int packetHeaderByteLength = 2 + 4 + 1 + 1;
```

Comment:
Please make this a final constant.
FYI you can also format the JavaDoc with HTML tags so it renders not just inline:

```java
    /**
     * Packet header length in bytes:
     * <ul>
     *   <li>2 for id</li>
     *   <li>4 for sample rate as int little-endian</li>
     *   <li>1 for bitDepth</li>
     *   <li>1 for channels</li>
     * </ul>
     */
```